docs/getting_started/js_utils.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+title: JavaScript Utilities
+parent: MediaPipe in JavaScript
+nav_order: 1
+---
+
+# JavaScript Utils for MediaPipe
+{: .no_toc }
+
+1. TOC
+{:toc}
+---
+
+## Utilities
+MediaPipe offers utilities for enabling users to use the MediaPipe Examples with ease, and conveniently integrate with existing code.                       
+The utilities MediaPipe currently has are ~
+* [Drawing Utilities][draw-npm],
+* [Camera Utilities][cam-npm], and
+* [Control Panel][ctrl-npm] Utilities.
+
+### Drawing Utilities
+These are for drawing on canvas using output from MediaPipe's solutions.                 
+Currently, this includes three functions ~
+1. *drawLandmarks*                                               
+```js
+drawLandmarks(canvasContext, landmarks, {
+            fillColor: fillColor,
+            color: color,
+            lineWidth: lineWidth
+        }
+```                  
+This function is for drawing landmarks.                                               
+The value for the `landmarks` parameter can be found using in the `results` array.
+
+2. *drawConnectors*
+```js
+drawConnectors(canvasContext, landmarks, LANDMARK_CONSTANTS, {
+    color: color,
+    lineWidth: lineWidth
+})
+```                        
+This is a function used for plotting connectors in various MediaPipe Solutions.                         
+The `LANDMARK_CONSTANTS` are specific for every solution, so you could choose these there.                           
+
+3. *drawRectangle*
+```js
+drawRectangle(canvasContext, {
+    width: width,
+    height: height,
+    xCenter: xCenter,
+    yCenter: yCenter,
+    rotation: rotation
+}, {
+    color: color
+})
+```
+This allows drawing of a rectangle, with a certain width, height and rotation given the coordinates of its location.
+                                                              
+### Camera Utilities
+This module has a `Camera` object that can be initialized and used easily, like below.
+```js
+// Our input frames will come from here.
+const videoElement =
+    document.getElementsByClassName('input_video')[0];
+const camera = new Camera(videoElement, {
+  onFrame: async () => {
+    // Send it to a demo, for example, hands
+    await hands.send({image: videoElement});
+  },
+  // Set the width of camera to be rendered
+  width: 1280,
+  // And set the height
+  height: 720
+});
+// Start the camera.
+// Note: It will ask the user for permission, and will error out if this is not given.
+camera.start();
+```
+### Control Panel Utilities
+To showcase and monitor the model in elegant ways, MediaPipe has this Control Panel module.
+This panel is also convenient to use and set up.
```

Comment:
I think we can import FPS from control panel utilities, too, right? Probably worth documenting that too.